internal/controller/storageclient_controller.go

```diff
@@ -286,6 +304,7 @@ func (r *storageClientReconcile) reconcileDynamicWatches() error {
 	}
 	// CRD doesn't exist in the cluster
 	if crd.UID == "" {
+		r.log.Info("VolumeGroupSnapshotClass CRD is not available", "crdName", VolumeGroupSnapshotClassCrdName)
```

Comment:
this'll add a negative log line in every reconcile and not sure why this is required.